viz.yaml

```diff
@@ -429,14 +420,11 @@ publish:
   -
     id: wu_story
     template: layout/templates/wu_story.mustache
-    depends: ["styles", "placeholder", "static_story_figure", 
-              "static_story_figure_mobile_left", 
-              "static_story_figure_mobile_right"]
+    depends: ["styles", "placeholder", "static_story_figure", "static_story_figure_mobile"]
     context:
       placeholder: "placeholder"
       pie_map_plot: "static_story_figure"
-      mobile_left_pie_plot: "static_story_figure_mobile_left"
-      mobile_right_pie_plot: "static_story_figure_mobile_right"
+      mobile_pie_map_plot: "static_story_figure_mobile"
```

Comment:
are there mustache template changes that reflect the replacement of `mobile_left_pie_plot` and `mobile_right_pie_plot` with `mobile_pie_map_plot`?

Reply:
Nope, they were added as a dependent to a mustache file. Those were simply images.

Reply:
Or the change is reflected in my next commit as you were right I did not commit the mustache file for whatever reason...